### PR TITLE
Update unit testing

### PR DESCRIPTION
**Pull request by westonruter**
Use a `install-wp-tests.sh` now bundled with `dev-lib` instead of using the one from WP-CLI.

Add support for `WP_VERSION=trunk`.

### CODE REVIEW

**Comment by fjarrett** [comment]
These changes caused our bootstrap file to break since `WP_TESTS_DIR` no longer has a trailing slash. So I had to add one in https://github.com/wp-stream/stream/commit/5f3946431658d70ff4538083cdde705b22047946

File: install-wp-tests.sh [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+fi
+
+DB_NAME=$1
+DB_USER=$2
+DB_PASS=$3
+DB_HOST=${4-localhost}
+WP_VERSION=${5-latest}
+
+WP_CORE_DIR=${WP_CORE_DIR-/tmp/wordpress}
+WP_TESTS_DIR=${WP_TESTS_DIR-${WP_CORE_DIR}/tests/phpunit}
```
Comment (westonruter) [inline install-wp-tests.sh]: @fjarrett so you're saying that this should instead be:

```diff
- WP_TESTS_DIR=${WP_TESTS_DIR-${WP_CORE_DIR}/tests/phpunit}
+ WP_TESTS_DIR=${WP_TESTS_DIR-${WP_CORE_DIR}/tests/phpunit/}
```

Comment (fjarrett) [inline install-wp-tests.sh]: @westonruter Yes :smiley:

Comment (westonruter) [inline install-wp-tests.sh]: Cool. Mind opening a PR and ensuring it works with Stream?

**Pull request merged**